oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,309 @@
+Owners were also responsible for the health of the component they
+owned both in the repository and in production; owners answered the
+pager when something went wrong.
+
+The role of the repository or project maintainer is both a community
+focused and a technical role[#]_.  Maintainers will be the public face
+of each component and should be welcoming and available.
+
+Non-technical responsibilities of maintainers include:
+
+* Effective communication with community members, other maintainers,
+  and project leadership
+* Disciplined management to ensure that maintenance work is done
+* Enforcement of the project's standards
+* The ability to solve or escalate issues facing users of the
+  component
+
+Technical responsibilities of maintainers include:
+
+* Understanding the current component architecture and implementation
+* Having a technical vision for the evolution of the component
+* The ability to assess the appropriateness of proposed changes to or
+  extensions of the component
+* The role of the component in the overall platform and how it
+  interacts with other components
+
+It is acceptable for a component to be maintained by a team or group
+as long as that arrangement does not reduce the reachablility of the
+maintainer or reduce accountability.
+
+Prerequisites
+-------------
+In order to be a repository maintainer, you must
+
+* Be an active member of the Open edX community who has contributed to
+  the project in the last 6 months.  The types of contributions that
+  are relevant will match those defined in the `core contributor
+  program`_
+* Abide by the `Open edX Code of Conduct
+  <https://openedx.org/code-of-conduct/>`_
+* Be a core contributor for the repository in question who has
+  demonstrated excellence in the role of core contributor.  From
+  time-to-time, in the case of a strong candidate and project need,
+  core contributor status and maintainer status may be conferred at
+  the same time.
+
+Bootstrapping
+-------------
+In order to ensure an orderly roll-out of the formal maintainer role,
+existing 2U staff in the owner role will transition into the role of
+maintainer for the components that they own.
+
+Community members who are acting in informal maintainer roles, will
+also be asked to become named maintainers.
+
+A list of all components without maintainers will be created and
+published.
+
+An asynchronous maintainer scrum-of-scrums will be created as a Slack
+channel.
+
+Standards for measuring the quality of repositories will be rolled out
+across all repositories.  These standards can benefit from work done
+at edX to measure `repository health <https://github.com/openedx/edx-repo-health>`_.
+
+Becoming a Maintainer
+---------------------
+After the bootstrapping phase, the Open edX project will need
+processes for adding additional maintainers, replacing maintainers,
+and ensuring that new components are maintained.
+
+**This section should be congruent with the Core Contributor process**
```

Comment:
I would either make one document the canonical doc, so that updates are only made in one place, or make both docs point to one another so it's clear to authors later on they should update in two places.

Reply:
> to become a maintainer, you must first be a core contributor with access to the relevant repo. 

@feanil this is already covered in the section `Prerequisites`

As far as following the CC process to be added as a maintainer, I'm going to let the comment thread below hash out.

Reply:
@feanil I've tried to clarify this point reinforcing what @sarina has already mentioned, if you are satisfied, please resolve.